types/eslint/index.d.ts

```diff
@@ -496,6 +496,8 @@ export class CLIEngine {
 
     static getErrorResults(results: CLIEngine.LintResult[]): CLIEngine.LintResult[];
 
+    static getFormatter(format?: string): CLIEngine.Formatter;
```

Comment:
I see the docs for `version` being static, but where is this documented as static? It looks to be an instance method [here](https://eslint.org/docs/developer-guide/nodejs-api#cliengine-getformatter).

Reply:
Seems like it is both an instance method and a `static` method. From that link:

> Note: Also available as a static function on CLIEngine

Reply:
Yes, information about static `getFormatter` can easily be overlooked.